content/WebAuthn/Supporting_U2F_or_FIDO2_Security_Keys_on_iOS_or_iPadOS/Security_Key_Compatibility.adoc

```diff
@@ -0,0 +1,8 @@
+== Security Key Compatibility
+
+iOS 13.3 opens up a large pool of supported security keys when using the Safari browser:
+
+image:table-1[]
```

Comment:
this doesn't render, I assume there's a missing .png here?

Reply:
Also: maybe this should be included as text rather than be a table included as an image?
there's also a blue border in the image that probably doesn't belong.

Reply:
That's right, there's a missing .png. The image is a horror in the sense that even when you zoom in, it will be really difficult to read. I will submit the table as an image, and change it out for a table if I get the time to create one.